src/ChatWindow/Room/Room.vue

```diff
@@ -514,6 +515,22 @@ export default {
 			this.scrollMessagesCount = 0
 			this.resetMessage(true, null, true)
 
+			const roomId = this.room.roomId
+			this.message = this.typedMessages[roomId] || ''
+			this.typedMessage = this.message
+			delete this.typedMessages[roomId]
+
+			const storedMessage = localStorage.getItem(roomId)
+
+			if (storedMessage) {
+				this.message = storedMessage
+				this.onChangeInput()
+			}
```

Comment:
I think we can remove this getting Items from local storage and the if block because i don't see anywhere anyitem being set to local storage. Remove those if no need.